### PR TITLE
Disable create instance button in case not all fields are required

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- Disable create instance button in case of errors
- update e2e test to check for this

**Related issue(s)**

See also https://github.com/kyma-project/kyma/issues/3766

### CODE REVIEW

**Comment by aerfio** [comment]
Use those lines in `helm-broker` config to check this PR:
```
testBundleUrl: process.env.HELM_BROKER_TEST_BUNDLE_URL
    ? process.env.HELM_BROKER_TEST_BUNDLE_URL
    : 'https://github.com/kyma-project/bundles/releases/download/latest/index-testing.yaml',
```

**Comment by aerfio** [comment]
It will be updated when https://github.com/kyma-project/bundles/pull/87 will be released

File: tests/ui-tests/commands/catalog.js [hunk 1]
```diff
@@ -18,9 +20,11 @@ module.exports = {
       const plan = `[name="selectedKind"]`;
       const additionalData = '#root_additionalData';
       const planName = '#root_planName';
+      const regExpData = '#root_onlyNumbersString';
       const modalCreate = `[${
         config.catalogTestingAtribute
       }="modal-confirmation-button"]`;
+      const disabledButton = '.is-disabled';
```
Comment (magicmatatjahu) [inline tests/ui-tests/commands/catalog.js]: `disabledButtonClass`

File: tests/ui-tests/commands/catalog.js [hunk 2]
```diff
@@ -18,9 +20,11 @@ module.exports = {
       const plan = `[name="selectedKind"]`;
       const additionalData = '#root_additionalData';
       const planName = '#root_planName';
+      const regExpData = '#root_onlyNumbersString';
```
Comment (magicmatatjahu) [inline tests/ui-tests/commands/catalog.js]: `regExpDataId`

File: tests/ui-tests/commands/catalog.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ module.exports = {
     instanceLabel,
     instanceAdditionalData,
     instancePlanName,
+    instanceRegExpData,
+    faultyRegExpData,
```
Comment (magicmatatjahu) [inline tests/ui-tests/commands/catalog.js]: Maybe call this like in config? `uncorrectRegExpData`

File: tests/ui-tests/commands/catalog.js [hunk 4]
```diff
@@ -9,6 +9,8 @@ module.exports = {
     instanceLabel,
     instanceAdditionalData,
     instancePlanName,
+    instanceRegExpData,
```
Comment (magicmatatjahu) [inline tests/ui-tests/commands/catalog.js]: Maybe call this like in config? `correctRegExpData`

**Pull request merged**